### PR TITLE
q47

### PR DESCRIPTION
**Pull request by Navleenkaursandhu**
q47:
image

### CODE REVIEW

File: Javascript/q47.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    .flex {
+      display: flex;
+      gap: 10px;
+    }
+  </style>
+
+  <script>
+    var val = 0;
+    function myFunc() {
```
Comment (Mantaseus) [inline Javascript/q47.html:22]: Please, name your functions more meaningfully. Use something like `incrementVal` or something

File: Javascript/q47.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    document.getElementById("demo0").innerHTML= val--;
```
Comment (Mantaseus) [inline Javascript/q47.html:23]: Fix indentation here

File: Javascript/q47.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    function myFunction() {
```
Comment (Mantaseus) [inline Javascript/q47.html:26]: Improve name please

File: Javascript/q47.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      document.getElementById("demo0").innerHTML= val++;
+    }
+  </script>
+</head>
+
+<body>
+  <div class="flex">
+  <button onclick="myFunc()">-</button>
+  <div id="demo0">0</div>
```
Comment (Mantaseus) [inline Javascript/q47.html:35]: Use better ID values

File: Javascript/q47.html (hunk 2 above)
Comment (Mantaseus) [inline Javascript/q47.html:23]: The question says "Minimum allowed value is 0". You do not have any protection against negative values

File: Javascript/q47.html (hunk 3 above)
Comment (Navleenkaursandhu) [reply Javascript/q47.html:26]: yes, done

File: Javascript/q47.html (hunk 4 above)
Comment (Navleenkaursandhu) [reply Javascript/q47.html:35]: done

**Review by Mantaseus — APPROVED** [review]

**Pull request merged**